### DISCUSSION TITLE
What is this "xxx" in the request log?

### DISCUSSION THREAD

**Discussion by a-h-abid** [fastify/fastify#3918] [Q&A]
Small question, what is this "xxx" thing that appears whenever a request is done? How can I change it or remove it?

**Comment by mcollina** [comment]
I don't know where that xxx comes from? 
Can you provide steps to reproduce? We often need a reproducible example, e.g. some code that allows someone else to recreate your problem by just copying and pasting it. If it involves more than a couple of different file, create a new repository on GitHub and add a link to that.

**Reply by a-h-abid** [reply]
Sorry for delay response.

These are the steps to reproduce.

```
npm install fastify-cli --global
fastify generate --lang=typescript myapp
cd myapp
npm install
npm install --save-dev pino-pretty
npm dev:start
```

![Screenshot from 2022-06-01 22-05-25--fastify](https://user-images.githubusercontent.com/2020773/171449900-073937f5-1c05-44bc-8943-8040c13fbbb9.png)

**Reply by mcollina** [reply]
what request are you sending?

**Reply by a-h-abid** [reply]
Simple GET request.
